src/config/env/index.ts

```diff
@@ -17,7 +18,7 @@ export const getEnv = memoize(
     BLOCKNATIVE_KEY: process.env.REACT_APP_BLOCKNATIVE_KEY,
     FORTMATIC_KEY: process.env.REACT_APP_FORTMATIC_KEY,
     PORTIS_KEY: process.env.REACT_APP_PORTIS_KEY,
-    ZAPPER_API_KEY: process.env.REACT_APP_ZAPPER_API_KEY,
+    ZAPPER_AUTH_TOKEN: encode({ str: `${process.env.REACT_APP_ZAPPER_API_KEY}:`, encoding: 'base64' }),
```

Comment:
I would prefer if we leave the env config exactly as it is in `process.env`, just for traceability, and add `ZAPPER_AUTH_TOKEN` in `src/config/constants/index.ts` as we do [here](https://github.com/yearn/yearn-finance-v3/blob/develop/src/config/constants/index.ts#L99)

Reply:
Agreed, much better as a constant. Refactored with https://github.com/yearn/yearn-finance-v3/pull/651/commits/8d6398acf7b55f38574034bf639eafa482fbcc66